src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -357,10 +429,22 @@ window.qBittorrent.DynamicTable ??= (() => {
                 actions[this.columns[i].name] = onMenuItemClicked;
             }
 
+            // add auto resize option
+            const autoResizeAnchor = document.createElement("a");
+            autoResizeAnchor.href = "#autoResizeAction";
+            autoResizeAnchor.textContent = "Auto resize";
+            const img = document.createElement("img");
+            img.src = "images/configure.svg";
```

Comment:
This shouldn't be used. This icon implies more configuration.
No icon here is OK IMO.